install.sh

```diff
@@ -312,7 +312,12 @@ printf "
 Service is ready and healthy!
 Adding admin user to Django webservice now.
 "
-sudo docker exec mathesar_service python manage.py createsuperuser --no-input --username "$superuser_username" --email "$superuser_email" 2> >(grep -vi warn)
+while !  sudo docker exec mathesar_service python manage.py createsuperuser --no-input --username "$superuser_username" --email "$superuser_email"
+do
+  read -r -p "Choose a different admin username [mathesar]: " superuser_username
```

Comment:
Please change the default username to `admin`

Reply:
With this fix, that won't work. I.e., this fix essentially just prompts a user to create a different user if the username is already taken. The username in question, in this case, is `admin`. We would assume that's already taken, since we still don't know how it's being created.

I suggest we remove this change, and merge the other changes to get those fixes in.